openlibrary/templates/books/custom_carousel.html

```diff
@@ -45,10 +48,16 @@
   <div class="book carousel__item">
     <div class="book-cover">
       <a href="$(url)">
-        <img class="bookcover"
-          width="130" height="200"
-          title="$('%s%s'%(title,byline))"
-          $img_attr="$(cover_url)"/>
+        $if cover_url:
+          <img class="bookcover"
+            width="130" height="200"
+            title="$('%s%s'%(title,byline))"
+            $img_attr="$(cover_url)"/>
+        $else:
+          <div class="carousel__item__blankcover">
+            <div class="carousel__item__blankcover--title">$:macros.TruncateString(title, 70)</div>
+            <div class="carousel__item__blankcover--authors">$:macros.TruncateString(', '.join(author_names), 30)</div>
```

Comment:
author_names is not always defined. 

I'm getting
UnboundLocalError: local variable 'author_names' referenced before assignment (falling back to default template)

this needs to be conditional.

Reply:
Great catch! I'll fix this up :+1:

Reply:
Done ✅